### ISSUE TITLE
Serge process crashed

### ISSUE THREAD

**Issue by IanMayo** [serge-web/serge#71]
An instance of Serge server had been live for about an hour.

When the game controller returned to the MS Windows command prompt, he saw that Serge had stopped, with this error reported 3 times:

> Error Command failed with exit code 134
> Info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Above that error was:
> FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed – Javascript heap out of memory.

There was also a list of 13 long error codes.

Initial googling suggests the issue relates to the amount of memory allocated to the Node process.

Will conduct further investigation, and also try to get the error codes.

**Comment by IanMayo** [comment]
Going onsite to work through permutations this morning.

Considering:
1. Work back through prior versions to find one were the error doesn't arise. Investigate changes made after that version - try to isolate damaging change
2. Try another version of Node, in case Node has become corrupted
3. Try changing the port used for Serge, in case some unexpected other service/app is pinging it
4. Try removing blocks of `server.js` (or other server-side scripts) to isolate offending capability
5. Generate some stats on the memory increase rate. See if it's reliably the same. Then see if/how it is impacted by connections from Admin and/or Player pages

**Comment by IanMayo** [comment]
Ok, the problem was that at run-time express-pouchdb does some more configuration, including downloading more components.

I diagnosed the problem as follows:
* I re-ran some prior releases and they were fine
* the memory leak was occurring before there were any serge connections, so I presumed it must be in the `server` package
* I worked through `server.js` commenting out different portions of text.  Finally I isolated the issue to this line:
![image](https://user-images.githubusercontent.com/1108513/69821179-d28a5a00-11fa-11ea-9df9-61bc3f92a94f.png)

It wasn't an option to leave this line out, since that's where the database connection is configured.

This particular snapshot had not been run on an Internet-connected PC, so there were missing components.  The Internet connections bouncing was causing memory leaks in PouchDb/Express/NodeJS.

I _spliced_ the desired `client` package onto a repo with a working `server` package and it worked fine.
